modules/analyzing-the-collector-logs.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * troubleshooting/retrieving-and-analyzing-the-collector-logs-and-pod-status.adoc
+:_module-type: CONCEPT
```

Comment:
```suggestion
:_content-type: CONCEPT
```
**nitpick:** We still have to update older content for this change but this is what the latest guidelines recommend suing.

Reply:
ack